src/solver/cvc5/cvc5_solver.hpp

```diff
@@ -70,6 +71,7 @@ class Cvc5Sort : public AbsSort
   bool is_string() const override;
   bool is_uninterpreted() const override;
   uint32_t get_bv_size() const override;
+  std::string get_ff_size() const override;
```

Comment:
Just to clarify: This is a std::string because the ff size can be arbitrarily large?

Reply:
Yes.